test/spmd/test_dynamo_spmd.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+import torch
+import torch.nn as nn
+import torch_xla
+import torch_xla.core.xla_model as xm
+import torch_xla.experimental.xla_sharding as xs
+import torch_xla.debug.metrics as met
+import unittest
+
+import test_xla_sharding_base
+
+
+class SimpleLinear(nn.Module):
+
+  def __init__(self):
+    super(SimpleLinear, self).__init__()
+    self.fc1 = nn.Linear(128, 128)
+    self.relu = nn.ReLU()
+    self.fc2 = nn.Linear(128, 1)
+    # Add an additional 1x1 layer at the end to ensure the final layer
+    # is not sharded.
+    self.fc3 = nn.Linear(1, 1)
```

Comment:
Is this due to the lack of output sharding propagation?

Reply:
yea, in this pr I tried to keep it that output is replicated. We can expand this after output sharding pr is ready.